### ISSUE TITLE
The update bot doesnt work very well 

### ISSUE THREAD

**Issue by jorgegalindocruces** [47deg/org#72]
Looks like the bot is committing new changes every day but it doesn't work properly because there is a lack of update information in the stars that are showing with the real ones.

**Comment by juanpedromoreno** [comment]
The issue seems to be in the script used by Travis:

https://travis-ci.org/47deg/org/jobs/465623267#L790

```bash
git push using:  https://47deg:[secure]@github.com/47deg/47deg.github.io.git master
To https://github.com/47deg/47deg.github.io.git
 ! [rejected]        1bd0462e8f9ccbfc07d5dd0729a349784f1c9b94 -> master (non-fast-forward)
error: failed to push some refs to 'https://47deg:[secure]@github.com/47deg/47deg.github.io.git'
hint: Updates were rejected because the tip of your current branch is behind
```

**Comment by purrgrammer** [comment]
I'll take a look :eyes:

**Comment by purrgrammer** [comment]
Should be fixed now, see [the last build](https://travis-ci.org/47deg/org)

**Comment by juanpedromoreno** [comment]
Thanks @purrgrammer  ❤️ !

**Comment by jorgegalindocruces** [comment]
This is happening again on the website (https://47deg.github.io/).

For example, Arrow has now 2400 stars and it appears only 1157

**Comment by juanpedromoreno** [comment]
Yeah, it seems this is failing again. any ideas what would be the cause  @purrgrammer ?

**Comment by purrgrammer** [comment]
This is a bit strange, since I can see the generated HTML code for Arrow with 2000+ stars but it seems GitHub is serving an old version for some reason. 

```html
<h2 data-reactid="98">arrow</h2>
<p data-reactid="99">Functional companion to Kotlin&#x27;s Standard Library</p>
<ul data-reactid="100">
    <li data-reactid="101"><span class="octicon octicon-code" data-reactid="102"></span><span data-reactid="103">Kotlin, Java</span></li>
    <li data-reactid="104"></li>
    <li data-reactid="105">
      <span class="octicon octicon-git-branch" data-reactid="106"></span>
      <span data-reactid="107">171</span></li>
    <li data-reactid="108"></li>
    <li data-reactid="109">
      <span class="octicon octicon-star" data-reactid="110"></span>
      <span data-reactid="111">2191</span></li>
```

I'm investigating the issue, will keep you posted.

**Comment by purrgrammer** [comment]
I think it may be caused by a missconfiguration of the repository, since I just checked and GitHub Pages is disabled for `org`:

![org](https://user-images.githubusercontent.com/42009830/53163427-56f5df80-35ce-11e9-8ea6-449525d30782.png)

I'm going to enable it to publish from the `docs` folder.

**Comment by purrgrammer** [comment]
I've discovered that the Pages feature in GitHub has changed a bit since we wrote `org`, and apparently for publishing a organization website [requires us to rename the repository](https://help.github.com/en/articles/user-organization-and-project-pages#user-and-organization-pages-sites).

For publishing this under 47deg.github.io, we'd need to create a repository named `47deg` (or rename this one) and host the website there. Org is now publishing to: https://47deg.github.io/org/

**Comment by calvellido** [comment]
Hi!  Just my two cents, as it is part of the 47º webiste, I think it would be much better if we set the domain for org under our own 47deg.com umbrella (opensource.47deg.com or something like that).

@justinelsberry is gonna transfer the domain to Google Domain soon, when that's done, if you agree we can go through this quickly.

**Comment by purrgrammer** [comment]
> @justinelsberry is gonna transfer the domain to Google Domain soon, when that's done, if you agree we can go through this quickly.

:+1: [Setting up a custom domain](https://help.github.com/en/articles/quick-start-setting-up-a-custom-domain) should be straightforward.

Changed the repo name  to `47deg.github.io`, since changes in the GitHub Pages feature force us to name it like this for being the org-wide site, you can see the updated website in https://47deg.github.io/ The rendered HTML still has "old" values, but it updates when receiving the new data from GitHub.

I'm going to trigger a rebuild to have the initial HTML updated.

**Comment by purrgrammer** [comment]
It seems that renaming the repository had an unintended consequence: it changed the Github Pages configuration to serve the website from `master` instead of the `docs` folder in `master`. Furthermore, I cannot change it back to the old configuration:

![setts](https://user-images.githubusercontent.com/42009830/53166062-99221f80-35d4-11e9-9633-a0e0381adb24.png)

I've contacted GitHub support to address this issue, seems like a bug on their side.

**Comment by calvellido** [comment]
I think it is because of the name, if it's the user or organization one, it must be served from the master branch:

https://help.github.com/en/articles/user-organization-and-project-pages

**Comment by purrgrammer** [comment]
So, if I understood correctly, we should use the `47deg.github.io` repository just to push the generated site from the `master` branch since GitHub Pages for organizations doesn't seem to support serving from the `docs` directory, am I right @calvellido ?

What I'm suggesting to do is:
 - Re-rename this repository to `org`
 - Create a new `47deg.github.io` repo
 - Configure the `org` repo to push the built site to `47deg.github.io` to the `master` branch as Github requires us to do

**Comment by calvellido** [comment]
While that would be a very clever thing that I would have never come up to, I find it a bit cumbersome still.

My proposal would be:

* Re-rename this repository to `org`
* Publish the site to its default location (`47deg.github.io/org` I guess)
* Point our website links to that new location
* When we are able to, set a custom domain for it under 47deg.com

Anyway, maybe the best option will depend on the number of sites beyond 47deg.com pointing to 47deg.github.io, thoughts @MaureenElsberry @jorgegalindocruces?

**Comment by MaureenElsberry** [comment]
I'm now just seeing this somehow 1.5 months later 😯

I'm super slammed with stuff right now, but have added a task for myself to review site-links. I'll keep ya posted and try to get to this next week, it's in the queue!
